tools/jdk/jdk_build_file.bzl

```diff
@@ -67,4 +67,6 @@ java_runtime(
         ":jdk-lib",
         ":jre",
     ],
+    version = ___VERSION___,
```

Comment:
nit: Just so we are consistent with other templates, could we use {RUNTIME_VERSION} as the placeholder. And then String.format() instead of String.replace() in local_java_repository/remote_java_repository.